proto/storage/compliance_operator_v2.proto

```diff
@@ -240,3 +240,16 @@ message ComplianceOperatorSuiteV2 {
 
   ComplianceOperatorStatus status = 4;
 }
+
+// Next Tag: 9
+message ComplianceOperatorRemediationV2 {
+  string id = 1; // @gotags: sql:"pk,id,type(uuid)"
+  string name = 2; // @gotags: search:"Compliance Remediation Name,hidden"
+  // TODO: Name or ID?
+  string compliance_check_result_name = 3; // @gotags: search:"Compliance Remediation Check Result Name,hidden"
+  bool apply = 4; // @gotags: search:"Compliance Remediation Applied,hidden"
```

Comment:
```suggestion
  bool apply = 4;
```

I don't think this needs to be searchable at this point.

Reply:
done